### DISCUSSION TITLE
Found a trivial typo in the 2.0 documentation.

### DISCUSSION THREAD

**Discussion by mk0218** [sqlalchemy/sqlalchemy#9024] [Usage Questions] [answered]
[In this section](https://docs.sqlalchemy.org/en/20/orm/extensions/asyncio.html#sqlalchemy.ext.asyncio.async_session.params.session)
```
from sqlalchemy.ext.asyncio import create_async_engine
from sqlalchemy.ext.asyncio import async_sessionmaker

async def main():
    # an AsyncEngine, which the AsyncSession will use for connection
    # resources
    engine = create_async_engine('postgresql+asyncpg://scott:tiger@localhost/')

    AsyncSession = async_sessionmaker(engine)

    async with async_session() as session:  # here!
        session.add(some_object)
        session.add(some_other_object)
        await session.commit()
```
Shouldn't the `async_session()` part in the 11st line be `AsyncSession()`?
I don't see any '`async_session`' imported or declared in this example.

**Comment by zzzeek** [comment, marked as answer]
yeah it's wrong, but also I am noticing just a few minutes ago that people are copying the pattern of "make a sessionmaker, then make the session, then throw it away" in blogs, which is not really the point of sessionmaker, so I am going to just update that quick
